### ISSUE TITLE
Change tax distribution of agfi to 6% buy and 6% sell

### ISSUE THREAD

**Issue by AsgaaardFjord** [aggregatedfinance/agfi-dao#22]
### Update 10.03.

After some thinking, i will update my proposal to the following structure:

Buy 6%
3% Treasury
3% Eth reflections/rewards

Sell 6%
3% Treasury
2% Eth reflections/rewards
1% burn

Since we should be going upwards, the holders will be rewarded a bit more from the buys and with a small majority the community wants to keep the burn tax, it will be included in the sell tax structure

I hereby request to modify the tax model of agfi:

**Currently**
BUY 5%:
4% Treasury
1% Operations

SELL 13%:
6% Treasury
1% Burn
3% ETH Rewards
3% Operations
### 
New proposal

Buy 6%
3% Treasury 
3% Eth reflections/rewards

Sell 6%
3% Treasury 
3% Eth reflections/rewards

**Why the change?**

Those taxes came from a time, where high tax were the meta and in a bull run, people didn't care. However nowadays you need to compete with a lot of other projects and taxes around 5% or 6% each way are accepted by the communities in general

**Why no burn?**
The burn tax was implemented by agfi to fight the inflation bug in the v1 hard-fork contract. This isn't the case anymore with v3. 
Burn feature is more of a marketing tool for meme project to trick people into thinking that burning has a direct correlation to the price. Burning tokens simply reduces the marketcap and the selling pressure. But to have a significant impact, a lot of tokens would need to be burned anyways. 

**Why no marketing/operation tax?**

It is true that every project needs funds to operate, for marketing an development. But with the current volume, those taxes would provide only a very small amount. 
In short-term it would be easier to do a new proposal to allocate funds from the treasury to a marketing wallet. 
When the machine is well oiled again, operation/marketing tax could be re-implemented.

**Comment by Cryptobuzz1** [comment]
I agree with lower taxes, however my opinion is I like 1% burn, will.always be a psychological investment point of view  to any new investor.

**Comment by LeoKing7777** [comment]
I think it’s important to have a higher tax for the treasury as that is our main operating model that measures our success. Maybe 4% treasury 1% reflections ?

**Comment by BIGDOGGYSTYLER** [comment]
The proposal of setting a 6% fee for both buying and selling is agreed upon, but the exact distribution of the fee may be open for debate. 1% Burn instead of 3% Eth reflections/rewards.

Sell/Buy 6%
3% Treasury
2% Eth reflections/rewards
1% Burn

In the scenario where Ed's chief aim is to enhance the treasury's financial resources through trading activities, it could be posited that the operational model may not necessitate an excessive degree of concern or scrutiny.

**Comment by Brenden-Brenden** [comment]
For now,  this tax structure looks the most appealing. As a large investor, the rewards are a big draw-in because of the anticipation of the volume that will be generated as AGFI grows in popularity. I do not believe that the burn is necessary because with some of the utilities that this project is moving towards, there will be buy backs anyway. The auto burn is an unnecessary feature at the moment. The balance of rewards to holders and funding the treasury is perfect with this even split, and honestly, this is incentivizing myself and others to add even more to our positions...Lower taxes, higher rewards, count me in

**Comment by AsgaaardFjord** [comment]
Buy 6%
3% Treasury
2% Eth reflections/rewards
1% burn

Sell 6%
3% Treasury
3% Eth reflections/rewards

or the other way around

**Comment by AsgaaardFjord** [comment]
After some thinking, i will update my proposal to the following structure:

Buy 6%
3% Treasury
3% Eth reflections/rewards

Sell 6%
3% Treasury
2% Eth reflections/rewards
1% burn

Since we should be going upwards, the holders will be rewarded a bit more from the buys and with a small majority the community wants to keep the burn tax, it will be included in the sell tax structure

**Comment by Cryptobuzz1** [comment]
These tax channels are very reasonable going forward, and as mentioned when volume turns up, they can be proposed again to the best interest of the project.

On this note the proposal has been approved, and will be moved to snapshot que in next 24 hours.

**Comment by Brenden-Brenden** [comment]
This makes sense to me, great job Asgaaard!

**Comment by Cryptobuzz1** [comment]
Result successful, new tax allocations have been changed on 16/03/23.

See here for results
https://snapshot.org/#/aggregatedfinance.eth/proposal/0x360adc36ba6e919145aa76a6a0c6da53a260b93dbcb7cec680d582df5832a501

Proposal now closed as completed
